libc-bottom-half/sources/sbrk.c

```diff
@@ -15,7 +15,7 @@ void *sbrk(intptr_t increment) {
         abort();
     }
 
-    uintptr_t old = __builtin_wasm_memory_grow(0, (uintptr_t)increment / PAGESIZE);
+    const uintptr_t old = __builtin_wasm_memory_grow(0, (uintptr_t)increment / PAGESIZE);
```

Comment:
Is there any value in adding a const here?

Reply:
Just to follow good coding style (in case if function will be extended).